Code.js

```diff
@@ -0,0 +1,193 @@
+var FREQUENCY = {
+  DAILY: 1,
+  WEEKLY: 2,
+  PAUSED: 3,
+};
+
+var SHEET_DATA = "Data";
+
+//return all participants from the Sheet_Data
+function getParticipants() {
+  var range = "A2:D";
+  var rangeValues = SpreadsheetApp.getActiveSpreadsheet()
+    .getSheetByName(SHEET_DATA)
+    .getRange(range)
+    .getValues();
+  var participants = [];
+
+  rangeValues.forEach((row) => {
```

Comment:
mmm... I didn't know google apps script supports arrow functions

Reply:
It's mostly ES6 from what i can gather. the editor hates it though, gets confused easily with indentation.